libraries/standard/http/src/http_client.c

```diff
@@ -682,40 +758,40 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
 
 /*-----------------------------------------------------------*/
 
-static HTTPStatus_t sendHttpHeaders( const HTTPTransportInterface_t * pTransport,
-                                     const HTTPRequestHeaders_t * pRequestHeaders )
+static HTTPStatus_t sendHttpData( const HTTPTransportInterface_t * pTransport,
+                                  const uint8_t * pBuffer,
+                                  size_t bufferLen )
```

Comment:
What do you think about having a struct that holds both the buffer and its length for consistency? Just thinking that it might be a little weird that MQTT and HTTP do those things differently. Maybe such types can be in a shared header file for both libraries under utilities?

Reply:
Having a structure in a common header simply for a buffer+len is excessive.